lib/internal/Magento/Framework/Filesystem/Driver/File.php

```diff
@@ -660,6 +660,9 @@ public function fileUnlock($resource)
      */
     public function getAbsolutePath($basePath, $path, $scheme = null)
     {
+        if (strlen($path) > 0 && $this->fixSeparator($path)[0] === '/') {
```

Comment:
That's a good point.  I need to review the changes I made here and see if there aren't others I should provide as a standalone fix.